utils/web3.ts

```diff
@@ -285,7 +283,7 @@ export const checkInterface = async (address: string, web3: Web3) => {
     isErc1271,
     isLsp0Erc725Account,
     isLsp1UniversalReceiver,
-    isLsp1UniversalReceiverDelegate,
+    // isLsp1UniversalReceiverDelegate,
```

Comment:
FMI, why is this one commented and not just removed ? Do we want to activate it again later ?

Reply:
Ahh it's a miss from me, let me change that thx